packages/react-dev-utils/README.md

```diff
@@ -57,7 +57,7 @@ module.exports = {
 ```
 
 
-#### `new ModuleScopePlugin(appSrc: string)`
+#### `new ModuleScopePlugin(appSrc: string, allowedPaths: [string])`
```

Comment:
Ditto, `allowedFiles`. And we should probably let them know it's optional `allowedFiles?: string[]`.